backend/test/test_data/claimEvent.js

```diff
@@ -0,0 +1,15 @@
+import { CID } from 'multiformats/cid'
+
+const claimTestEvent = {
+  id: 87871691,
+  client: 3297188,
+  provider: 3254061,
+  pieceCid: CID.parse('baga6ea4seaqk5bmtyfomjprov2jjtto7ffhxuusfouhn3fdu7l6gl6yemhzocoi'),
+  pieceSize: 34359738368,
+  termMin: 1008000,
+  termMax: 1267200,
+  termStart: 4620803,
+  sector: 102905
+}
+
```

Comment:
```suggestion
Object.freeze(claimTestEvent);
```

We can also freeze test objects to make sure data is not mutated in the tests.